src/features/workspace/backups.md

```diff
@@ -6,25 +6,29 @@ Creating backups is a `Pro` feature, but **all** plans can restore backups.
 
 ## Create
 
+---
+
 > **PRO FEATURE**
 
 You create backups directly from the `Projects` tab of your `Workspace` in the 3-dots menu.
-It is possible to backup any project including the currently selected one.
+It is possible to backup any Project including the currently selected one.
 
-<img alt="Backups creation" src="/_images/backup_creation.png" no-shadow/>
+<img alt="Backups creation." src="/_images/backup_creation.png" no-shadow/>
 
 Once the backup is done, it will be available in the `Backups` tab. If your instance is remote, you can `Download` the backup to your local machine. Otherwise, you can click on the 3-dots menu to get the path on disk if you wish to copy it directly.
 
-<img alt="Backups list" src="/_images/backups.png" no-shadow/>
+<img alt="Backups list." src="/_images/backups.png" no-shadow/>
 
 ## Restore
 
+---
+
 Restoring a backup is usually done via the `Import` button in the `Projects` tab.
 You will need to upload a backup file to the instance to perform the restore.
-Restoring always creates a new project, that is why you also need to enter a unique name.
+Restoring always creates a new Project, that is why you also need to enter a unique name.
 
-<img alt="Import project" src="/_images/backup_import.png" width="500" center/>
+<img alt="Import project." src="/_images/backup_import.png" width="500" center/>
```

Comment:
I dont know what the best practices are for alt text on images, do they usually include a dot?

Reply:
Not sure. Being the child of English teaching parents has just kicked in haha.

Reply:
its correct